src/Microsoft.Identity.Web.GraphServiceClientBeta/Microsoft.Identity.Web.GraphServiceClientBeta.csproj

```diff
@@ -17,7 +17,12 @@
     <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="8.0.0" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' != 'net8.0'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'net9.0'">
+    <PackageReference Include="Microsoft.Graph.Beta" Version="5.56.0-preview" />
+    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="9.0.0-preview.2.24128.5" />
```

Comment:
9.0.0-preview.2.24128.5 is in so many places. When we want to update to the nice preview, it would be nice to have just a single var to update

Reply:
Should we use wildcard - 9.0.0-* - so it's easier to be up-to-date (don't have to update each time a new preview is released) and may catch some bugs/behavior changes in .NET?

Reply:
Last year, that did not work, we had to specify the version, especially for the builds. :(

Reply:
New previews are monthly, so it's doable, but not ideal

Reply:
Added var to control preview versions. Thanks Keegan, Peter and Jenny!